version.txt

```diff
@@ -1 +1 @@
-1.7.1
+1.8.0
```

Comment:
There's already backwards incompatible changes on `main` so the next bump needs to be 2.0.0.

I'd appreciate if we could get https://github.com/taskcluster/taskgraph/pull/56 into the release as well :).

I already have a patch that does the bump and adds a migration guide so probably best to just drop this commit from your PR

Reply:
Er, unless this is still a ways from being ready and we release `2.0` first, in which case you can keep this around bumping to `2.1.0`

Reply:
Oh right! Thanks for reminding me that! I updated the changelog with your recent changes. Let's see which PR lands last to cut the 2.0.0 release 🙂

Reply:
Oh I meant I have a local stack that does it which I haven't submitted to a PR yet, so was planning to just do it as a standalone PR once ready :p, but we can do it as part of this too :)